core/config/project_settings.cpp

```diff
@@ -1506,6 +1506,9 @@ ProjectSettings::ProjectSettings() {
 
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "rendering/textures/canvas_textures/default_texture_filter", PROPERTY_HINT_ENUM, "Nearest,Linear,Linear Mipmap,Nearest Mipmap"), 1);
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "rendering/textures/canvas_textures/default_texture_repeat", PROPERTY_HINT_ENUM, "Disable,Enable,Mirror"), 0);
+	
+	GLOBAL_DEF_BASIC("rendering/2d/snap/snap_2d_transforms_to_pixel", false);
+	GLOBAL_DEF_BASIC("rendering/2d/snap/snap_2d_vertices_to_pixel", false);
```

Comment:
Thanks for the review. My proposal is to place these settings in the default (i.e. non-Advanced) section of the Project Settings, because other pixel-art-related options already are too. Adding a GLOBAL_DEF_BASIC here seemed to be the way to do that, as that's consistent with other properties. Did I misunderstand? What's the correct way to mark a property as "This should be visible even when Advanced is not toggled on?"

Reply:
I see, thanks!